### PR TITLE
Fix migrations for mysql and add tests

### PR DESCRIPTION
**Pull request by elainearbaugh**
Fix issue with a unique column with no max length.

Add tests for MySQL and update Jenkins to run the tests. Needed to get rid of the Russian in the tests because of a [MySQL issue](http://stackoverflow.com/questions/2108824/mysql-incorrect-string-value-error-when-save-unicode-string-in-django)

@mahendra @BlaineAtAffirm

### CODE REVIEW

**Comment by elainearbaugh** [comment]
trigger build changed jenkins job

**Comment by elainearbaugh** [comment]
trigger build changed it again

File: cabot/cabotapp/tests/tests_basic.py [hunk 1]
```diff
@@ -180,7 +180,7 @@ def fake_calendar(*args, **kwargs):
 
 
 def throws_timeout(*args, **kwargs):
-    raise requests.RequestException(u'фиктивная ошибка innit')
```
Comment (mahendra) [inline cabot/cabotapp/tests/tests_basic.py]: why was this required?

File: cabot/cabotapp/tests/tests_basic.py [hunk 2]
```diff
@@ -280,7 +280,7 @@ def test_timeout_handling_in_jenkins(self):
         checkresults = self.jenkins_check.statuscheckresult_set.all()
         self.assertEqual(len(checkresults), 1)
         self.assertFalse(self.jenkins_check.last_result().succeeded)
-        self.assertIn(u'Error fetching from Jenkins - фиктивная ошибка innit',
```
Comment (mahendra) [inline cabot/cabotapp/tests/tests_basic.py]: same ..

File: cabot/cabotapp/tests/tests_basic.py (hunk 1 above)
Comment (elainearbaugh) [inline cabot/cabotapp/tests/tests_basic.py]: mysql 3 byte limit on unicode characters (I guess I could change the charset to utf8mb4 but that doesn't seem super necessary), see http://stackoverflow.com/questions/2108824/mysql-incorrect-string-value-error-when-save-unicode-string-in-django

Comment (mahendra) [inline cabot/cabotapp/tests/tests_basic.py]: 👍

**Comment by elainearbaugh** [comment]
trigger build
